pitop/camera/camera.py

```diff
@@ -59,6 +64,24 @@ def __init__(self,
         self.__process_image_thread = Thread(target=self.__process_camera_output, daemon=True)
         self.__process_image_thread.start()
 
+        self.name = name
+        Stateful.__init__(self)
+        Recreatable.__init__(self, config_dict={
+            "index": index,
+            "resolution": resolution,
+            "camera_type": camera_type.value if isinstance(camera_type, Enum) else camera_type,
+            "path_to_images": path_to_images,
+            "format": format,
+            "name": self.name,
+        })
+
+    @property
+    def own_state(self):
```

Comment:
Not sure that I like this property name

Reply:
Yeah, I'm not a fan of it either, can we not just call it `state`? Or `self_state`